etest/docker/image.py

```diff
@@ -9,12 +9,17 @@
 from etest.docker import common
 
 
+def build(*args, **kwargs):
+    """Build a docker image."""
+    return common.CLIENT.images.build(*args, **kwargs)
```

Comment:
I'm starting to think we should drop our interface and inject this.  It'd certainly make for less code to maintain and make things immensely more testable.

Reply:
It makes migrating apis easier, but thats about it. We can talk on matrix about a better implementation tho.